stock_by_warehouse_stock/views/stock_move_views.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+     <record id="view_stock_move_operations" model="ir.ui.view">
+        <field name="name">view_stock_move_operations_absa</field>
+        <field name="model">stock.move</field>
+        <field name="inherit_id" ref="stock.view_stock_move_operations"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='picking_type_entire_packs']" position="after">
+                <div class="alert alert-primary font-weight-bold text-center" role="alert" groups="stock.group_stock_multi_warehouses">
+                    <p class="mb-1">Stock Availability from the warehouse: <field name="warehouse_id"/> is
+                        <field name="warehouses_stock_recompute" nolabel="1" colspan="1" widget="toggle_button"/>
+                    </p>
+                    <p class="mb-0">
+                        <field name="warehouses_stock_locations" widget="product_available_stock_locations"/>
```

Comment:
I expected the same widget with an option. Is that possible?

Reply:
Done :smile: